lib/models/file-info.js

```diff
@@ -84,11 +86,13 @@ FileInfo.prototype.render = function() {
       context = this.templateVariables;
   if (!this.rendered) {
     this.rendered = readFile(path).then(function(content){
-      try {
-        return processTemplate(content.toString(), context);
-      } catch (error) {
-        return content.toString();
-      }
+      return lstat(path).then(function(fileStat) {
+        if (isBinaryFile(content, fileStat.size)) {
```

Comment:
There already is one, see https://github.com/ember-cli/ember-cli/blob/master/tests/unit/models/file-info-test.js#L69